### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Sparktan24**
# Project requirements
- Add any missing JSX elements to your project (without adding extra functionality)
- Your final design should match Zeplin's design (this includes all elements + the same styling)
- You may use the CSS rules detailed in Zeplin to style the website.

For more info visit: https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md

### CODE REVIEW

File: src/components/AddBook.jsx [hunk 1]
```diff
@@ -2,15 +2,19 @@ import React, { useState, useEffect } from 'react';
 import { useDispatch } from 'react-redux';
 import { nanoid } from 'nanoid';
 import { addBook, getBooks } from '../redux/books/booksSlice';
+import '../styles/AddBook.css';
 
 const AddBook = () => {
   //  const { books } = useSelector((store) => store.books);
```
Comment (AnselemOdims) [inline src/components/AddBook.jsx]: - [x] Please kindly remove all commented code from your codebase as it is not a good practice.

File: src/index.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
```
Comment (AnselemOdims) [inline src/index.css:1]: - [x] Good job including one of the fonts, but kindly ensure all fonts used in the design are used in the implementation. These fonts are available on Google Fonts.

File: src/styles/AddBook.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  padding: 0.5rem 1rem;
+  color: #888;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  border-top: 1px solid rgb(222, 222, 222);
+  border-left: 1px solid rgb(222, 222, 222);
+  border-right: 1px solid rgb(222, 222, 222);
+  font-family: Montserrat, sans-serif;
+}
```
Comment (AnselemOdims) [inline src/styles/AddBook.css]: - [x] Good job styling this, but please kindly ensure that is spacing below the form as specified in the design.

### Your Design
image

### Original Design
image

File: src/styles/BooksList.css [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.books-container {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
```
Comment (AnselemOdims) [inline src/styles/BooksList.css:5]: - [ ] At the moment, there is a horizontal scroll on your page which is likely caused by the extended width of the containers. Please kindly remove the horizontal scroll as it does not give a good user experience.

image

File: src/components/NavBar.jsx [hunk 5]
```diff
@@ -1,17 +1,22 @@
 import { Link } from 'react-router-dom';
-import styles from '../styles/NavBar.module.css';
+import '../styles/navBar.css';
+import { HiOutlineUserCircle } from 'react-icons/hi';
+//  import styles from '../styles/NavBar.module.css';
```
Comment (AnselemOdims) [inline src/components/NavBar.jsx]: - [x] Please kindly remove this commented code

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Sparktan24 , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by christianbiring1 — APPROVED** [review]
Hi @Sparktan24 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Good looking UI :art: 
- [ ] Descriptive PR `title` and `summary` :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**